src/setfit/trainer.py

```diff
@@ -372,22 +373,37 @@ def train(
                 show_progress_bar=True,
             )
 
-    def evaluate(self):
-        """Computes the metrics for a given classifier."""
+    def evaluate(self, **evaluate_kwargs):
```

Comment:
IMO this would be better like this:

`    def evaluate(self, evaluate_kwargs):`

So without the `**`. The reason is that if there are the `**` then evaluate_kwargs cates 
all other function params. What if we want to extend the evaluate function to also pass 
args to someting else in the function? How do we then add an other param for that?

Reply:
Or just remove the metric_kwargs as suggested by @lewtun

Reply:
Thanks for the references! Maybe it's better to simply remove kwargs and leave the callable option, to avoid doing two different things at once in this PR

Reply:
Yes, I agree with removing kwargs in this PR - I'd like to revisit this once `evaluate` has implemented a new API for how one defines metrics with runtime args (like `f1` score). These will soon be defined entirely by `evaluate.load()` and that should make it easy to pass everything at once to the `metric` arg